pyproject.toml

```diff
@@ -11,8 +11,8 @@ authors = [
 requires-python = ">=3.9"
 
 dependencies = [
-  "torch ~= 2.0.1",
-  "torchvision ~= 0.15.2",
+  "torch >= 2.0",
+  "torchvision >= 0.15",
```

Comment:
I would make this:
```
torch >= 2.0.1
torchvision >= 0.15.2
```

Reply:
Actually that make not work correctly. Let me lookup the syntax again.

Reply:
Actually, maybe that does work? I think it's the `~` and `^` specifiers that affect major/minor/patch updating. See: https://python-poetry.org/docs/dependency-specification/

Reply:
Yes, `>=2.0.1` should work. 

However, we don't use poetry in MART.

Reply:
Fixed in 7b852d493db616ef81ca8cf4a459a5abf9b06b18